packages/store/src/Storage.sol

```diff
@@ -7,13 +7,20 @@ import { Bytes } from "./Bytes.sol";
 import { Memory } from "./Memory.sol";
 import "./Buffer.sol";
 
+/**
+ * @dev This implementation is very gas efficient.
```

Comment:
do we need this comment?

Reply:
Haha no, I think I just changed an old comment saying something like "not very gas efficient" and moved it from a function to the top of the lib